src/content/api/module-variables.md

```diff
@@ -131,6 +131,34 @@ The internal chunk loading function. Takes two arguments:
 - `chunkId` The id for the chunk to load.
 - `callback(require)` A callback function called once the chunk is loaded.
 
+T> You can override this function to customize retry behavior.  For example:
+
+```js
+const cdns = [
+	"https://cdn1.example.com/",
+	"https://cdn2.example.com/",
+	"https://cdn3.example.com/",
+];
+
+let currentCdn = 0;
+
+const original = __webpack_chunk_load__;
+__webpack_chunk_load__ = (id) => {
```

Comment:
Alignment is off, please fix. I also think this overrides webpack's public path from the start. Need to note it somewhere. I means it doesn't try original public path before starting going to cdns array.

Reply:
@EugeneHlushko using a linter without setting up a formatter to automatically fix pesky issues is extremely rude to contributors

Reply:
`lint:markdown` doesn't complain about the alignment anyway, I don't know what you're talking about.  Please be more considerate of developers' time.